src/tools/clippy/tests/ui/rename.rs

```diff
@@ -13,9 +13,8 @@
 #![allow(clippy::disallowed_methods)]
 #![allow(clippy::disallowed_types)]
 #![allow(clippy::mixed_read_write_in_expression)]
-#![allow(clippy::manual_find_map)]
 #![allow(clippy::manual_filter_map)]
-#![allow(unpredictable_function_pointer_comparisons)]
+#![allow(clippy::manual_find_map)]
```

Comment:
This file is supposed to be auto-generated (it has a comment at the top saying so), but most people seem to edit it manually. I used the generator though, that's why so many lines are changed in this file.